main/py-six/APKBUILD

```diff
@@ -20,6 +21,12 @@ build() {
 	python3 setup.py build
 }
 
+build() {
```

Comment:
I believe you meant to write check() ... :smile:

Reply:
Eh, true.

Reply:
Done, forcepushed new version.